.go-version

```diff
@@ -1 +1 @@
-1.22.2
+1.22
```

Comment:
This pins the go-version to the latest golang language version i.e. `v1.N` instead of the patch version `v1.N.P`.. 
This makes sure we are always building image with the latest patch for the golang version and does not need regular updates to pick the right patch version.